Formula/nginx-full.rb

```diff
@@ -246,9 +247,14 @@ def install
       args << "--add-module=#{HOMEBREW_PREFIX}/share/ngx-devel-kit"
     end
 
+    # Njs module configuration path needs to be appended with "/nginx" compared to other modules
+    if build.with?("njs-module")
+      args << "--add-module=#{HOMEBREW_PREFIX}/share/njs-nginx-module/nginx"
+    end
+
     # Third Party Modules
     self.class.third_party_modules.each_key do |name|
-      if build.with? "#{name}-module"
+      if build.with? "#{name}-module" && name != "njs"
```

Comment:
This is not necessary, it is necessary to correct in `njs-nginx-module` set subdir.